### PR TITLE
Add document on how to link to the UCRT correctly

### PR DESCRIPTION
**Pull request by njsmith**
This is the text I sent to our contacts at MS, and that they say is
correct.

These files need a full reorganization pass at some point, but I'm in a
rush and wanted to get this text up now that they've signed off on it,
so for now I've just wedged it in and we can fix it up later :-)

### CODE REVIEW

**Comment by techtonik** [comment]
Good analysis. It could be improved, because it took more than 5 minutes to get the idea. Still need more time to understand how "API sets" actually work and how it plays with licensing issues.

**Comment by njsmith** [comment]
I guess I figured that for the audience of 3-5 people who actually care
about implementing UCRT linkage in the mingw-w64 runtime, clarity and
context were more important than terseness :-) But if you're in a hurry the
TL;DR at the bottom sums it up.
Any licensing issues are orthogonal to the API sets issues -- the API sets
stuff basically comes down to "instead of linking to _this_ DLL like you
might naively think, you should link to _that_ DLL that's sitting next to
it and has identical distribution/licensing".

On Mon, Jan 11, 2016 at 8:25 PM, anatoly techtonik <notifications@github.com

> wrote:
> 
> Good analysis. It could be improved, because it took more than 5 minutes
> to get the idea. Still need more time to understand how "API sets" actually
> work and how it plays with licensing issues.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mingwpy/mingwpy.github.io/pull/23#issuecomment-170785899
> .

## 

Nathaniel J. Smith -- http://vorpus.org

**Comment by carlkl** [comment]
I will add my incomplete list of missing symbols for mingw-w64 binaries later:
- printf/scanf function family are now inlined in the MSVC header files and not exported anymore by the UCRT DLLs 
- some function names have been changed or are simply missing
